src/content/contributing/code-review/_index.en.md

```diff
@@ -6,44 +6,50 @@ weight: 30
-wherever relevant. This guide collects the most important highlights of the
-K8s process and adds Submariner-specific extensions.
+Submariner follows the [Kubernetes Code Review Guide][kube code review guide] wherever relevant. This guide collects the most important
+highlights of the K8s process and adds Submariner-specific extensions.
 
-### Two non-author approvals required
+### Two non-author Committer approvals required
 
-Pull Requests to Submariner require two non-author code review approvals.
-
-At least one approval must be from a Committer to the relevant part of the code
-base, as defined by the CODEOWNERS file at the root of the repository.
+Pull Requests to Submariner require two approvals from a Committer to the relevant part of the code base, as defined by the CODEOWNERS file
+at the root of each repository and the [Community
+Membership/Committers](http://localhost:1313/contributing/community-membership/#committers) process.
 
 ### No merge commits
 
 Kubernetes recommends [avoiding merge commits][merge commits].
 
-Use `git fetch` and `git rebase` to avoid them.
+With our current GitHub setup, pull requests are liable to include merge commits temporarily. Whenever a PR is updated through the UI,
+GitHub merges the target branch into the PR. However, since we merge PRs by either squashing or rebasing them, those merge commits
+disappear from the series of commits which ultimately ends up in the target branch.
 
 ### Squash/amend commits into discrete steps
 
 Kubernetes recommends [squashing commits using these guidelines][squashing].
 
 After a review, prepare your PR for merging by squashing your commits.
 
-All commits left on your branch after a review should represent meaningful
-milestones or units of work. Use commits to add clarity to the development and
-review process.
+All commits left on your branch after a review should represent meaningful milestones or units of work. Use commits to add clarity to the
+development and review process. Keep in mind that smaller commits are easier to review.
 
 Before merging a PR, squash the following kinds of commits:
 
 * Fixes/review feedback
 * Typos
 * Merges and rebases
 * Work in progress
-* Aim to have every commit in a PR compile and pass tests independently if you
-  can, but it's not a requirement.
+* Aim to have every commit in a PR compile and pass tests independently if you can, but it's not a requirement.
+
+### Address code review feedback with new commits
+
+When addressing review comments, as a general rule, push a new commit instead of amending to the prior commit as the former makes it easy
+for reviewers to determine what changed.
+
+But remember to squash them into the appropriate commit before merging. To make that simpler, commit with `git commit --fixup`, with the
+appropriate commit hash: this will keep your review-prompted changes as separate commits, but automatically select them for squashing with
+`git rebase --autosquash -i`.
```

Comment:
Not trying to nit but expanded the content a bit - hope it's all accurate.

```suggestion
To avoid cluttering the git log, squash the review commits into the appropriate commit before merging. The committer can do this in GitHub via the "Squash and merge" option.  However you may want to preserve other commits, in which case, squashing will need to be done manually via the Git CLI. To make that simpler, you can commit the review-prompted changes with `git commit --fixup` with the
appropriate commit hash. This will keep them as separate commits and, when you later rebase, specify the `--autosquash` option (that is `git rebase --autosquash -i`) to automatically select them for squashing.
```